providers/class.two-factor-fido-u2f.php

```diff
@@ -130,6 +114,23 @@ public function get_label() {
 	 * @since 0.1-dev
 	 */
 	public function login_enqueue_assets() {
+		
+		wp_register_script(
+			'fido-u2f-api',
```

Comment:
Looks like this is also referenced here:

https://github.com/WordPress/two-factor/blob/2b4ce5cd41a8859f5524f651493151baa61b7683/providers/class.two-factor-fido-u2f-admin.php#L75-L81

so we need to make sure this script is registered before it is enqueued there.

Reply:
Ah yea, made a change to the PR to register on `admin_enqueue_scripts`. But when does [login_enqueue_assets](https://github.com/WordPress/two-factor/blob/f8d2fda9c4f381c4242e66abf62486e47116d2ef/providers/class.two-factor-fido-u2f.php#L132) get called?